src/web-index/webIndex.ts

```diff
@@ -120,10 +117,31 @@ export function createWebIndex(ctx: Context): IWebIndexInterface {
         fileContents = pluginResponse.fileContents;
       }
 
+      fileContents = fileContents.replace(/\$import\('(.+?)'\)/g, (_, relPath: string) => {
```

Comment:
One more thing here, we should allow double quotes too.

Reply:
Also, would it make sense to call it "resolve" instead ?

Reply:
`resolve` instead of `import`? - I am indifferent

Reply:
I didn't want to include double quotes because of this:
![image](https://user-images.githubusercontent.com/28601907/71783030-695bea80-2fe1-11ea-91f9-7cb3333855dc.png)